src/main/java/tc/oc/pgm/rotation/VotingPool.java

```diff
@@ -0,0 +1,91 @@
+package tc.oc.pgm.rotation;
+
+import java.util.*;
+import org.bukkit.configuration.ConfigurationSection;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.map.PGMMap;
+
+@SuppressWarnings("unchecked")
+public class VotingPool extends MapPool {
+
+  // Number of maps in the vote, unless not enough maps in pool
+  private static final int MAX_VOTE_OPTIONS = 5;
+
+  private final double DEFAULT_WEIGHT;
```

Comment:
Would be nice if we gave `pgm.premium` double weight, possibly configurable.

Reply:
This weight is a map weight, not a vote weight.

If there are 10 maps, the default weight is 0.1 (10% chance)